lib/src/installer/completion_installation.dart

```diff
@@ -226,18 +226,16 @@ class CompletionInstallation {
     );
   }
 
-  String get _shellRCFilePath =>
-      _resolveHome(configuration!.shellRCFile, environment);
-
   /// Write a source to the completion global script in the shell configuration
   /// file, which its location is described by the [configuration].
   @visibleForTesting
   void writeToShellConfigFile(String rootCommand) {
     final configuration = this.configuration!;
+    final shellRunCommandFile = configuration.shell.runCommandFile();
 
     logger.info(
       'Adding dart cli completion config entry '
-      'to $_shellRCFilePath',
+      'to ${shellRunCommandFile.path}',
```

Comment:
What good is having `run command` doing here? Most people know those files as RC files. Unnecessary change.